module/apps/actor-importer-dialog.js

```diff
@@ -99,6 +99,19 @@ export class CoC7ActorImporterDialog extends Dialog {
   }
 
   submit (button) {
+    if(button.cssClass=="getExampleNow"){
```

Comment:
`if (button.cssClass === 'getExampleNow') {`
Add a space either side of the brackets, change == to === and use ' instead of " as per the Standard JS format